Sources/Fullscreen/Explore/ExploreView.swift

```diff
@@ -10,6 +10,10 @@ import FinniversKit
 public protocol ExploreViewDelegate: AnyObject {
     func exploreViewDidRefresh(_ view: ExploreView)
     func exploreView(_ view: ExploreView, didSelectItem item: ExploreCollectionViewModel, at indexPath: IndexPath)
+    func exploreViewRecommendations(_ adRecommendationsGridView: ExploreView, didSelectItemAtIndex index: Int, withId: String)
+    func exploreViewRecommendations(_ adRecommendationsGridView: ExploreView, willDisplayItemAtIndex index: Int)
+    func exploreViewRecommendations(_ adRecommendationsGridView: ExploreView, didScrollInScrollView scrollView: UIScrollView)
```

Comment:
This method seems to be unused and can be removed

Reply:
I think I'm supposed to use this for tracking, I just haven't gotten to it yet 🤔